<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailPhoBo</class>
 <widget class="QMainWindow" name="DetailPhoBo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chi tiết món Phở Bò</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>../img/phobo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>Phở Bò</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_desc">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>
Phở bò là món ăn truyền thống nổi tiếng của Việt Nam, với nước dùng trong, thơm mùi quế hồi, bánh phở mềm và thịt bò tươi ngon. Đây là món ăn sáng được yêu thích trên khắp cả nước.
     </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ingredient_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thành phần:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ingredients">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>
Bánh phở, thịt bò, nước dùng xương bò, hành lá, rau thơm, quế, hồi, gừng, hành tây, chanh, ớt.
     </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Quay về</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
        background-color: #1e1e1e;
        color: white;
        border-radius: 10px;
        font-size: 14px;
     </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>